### PR TITLE
Debug entry_stop

### PR DESCRIPTION
**Pull request by HenningSE**
This PR will fix the issue reported by @mhliu0001 here: https://github.com/XENONnT/fuse/pull/159#discussion_r1529325586

### CODE REVIEW

**Review by ramirezdiego — APPROVED** [review]
👍

**Comment by mhliu0001** [comment]
Just to ensure, does the `entry_stop` include the edge?

**Comment by ramirezdiego** [comment]
> Just to ensure, does the `entry_stop` include the edge?

No. Events up to this value should be included, excluding the value itself.

File: fuse/plugins/micro_physics/input.py [hunk 1]
```diff
@@ -404,11 +404,17 @@ def _load_root_file(self):
 
             if self.entry_stop is not None:
                 stop_index = np.argmin(all_eventids["eventid"] < self.entry_stop)
```
Comment (mhliu0001) [inline fuse/plugins/micro_physics/input.py:406]: Then this line is not correct?

Comment (ramirezdiego) [reply fuse/plugins/micro_physics/input.py:406]: True 😂

**Pull request merged**